hypervisor/arch/x86/guest/vmcall.c

```diff
@@ -105,6 +106,24 @@ static const struct hc_dispatch hc_dispatch_table[] = {
 		.permission_flags = GUEST_FLAG_SECURE_WORLD_ENABLED},
 };
 
+uint16_t allocate_dynamical_vmid(struct acrn_vm_creation *cv)
+{
+	uint16_t vm_id;
+	struct acrn_vm_config *vm_config;
+
```

Comment:
Shouldn't you check for a duplicate name here, maybe inside the spinlock if necessary, and return ACRN_INVALID_ID if there already is a VM with that name?

Reply:
this api only for post-launched VMs, and need to do the check for pre-launched VMs